site/en/blog/pwa-manifest-id/index.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Uniquely identifying PWAs with the web app manifest id property
+authors:
+  - petelepage
+date: 2021-09-15
+description: |
+  There's a new optional `id` property as part of the web app manifest spec,
+  that allows you to explicitly define the identifier used for your PWA.
+  Adding the `id` property to the manifest removes the dependency on the
+  `start_url` or the location of the manifest, and makes it possible for
+  them to be updated in the future.
+tags:
+  - progressive-web-apps
```

Comment:
We have a bug for this right now but you can tag this with `- chrome-96`, even if it's in the future. (If it causes the build to crash before you get to this, try rebasing or just leave it commented out.)

Reply:
Done